tests/backend-storage/src/test/utils/Helpers.ts

```diff
@@ -37,6 +38,14 @@ export async function queryAndAssertMetadata(
   expect(metadata).to.deep.equal(expectedMetadata);
 }
 
+export async function queryAndAssertHeaders(
```

Comment:
the function seems to only assert content type. I think a better name would be `queryAndAssertContentType`

Reply:
contentEncoding is the only header I have implemented, my idea was to add every new header that's implemented. I couldn't find a way to get a list of Typescript interface members at run time.

Reply:
you're right, I renamed the helper